### ISSUE TITLE
Quest "Cormath the Courier" shows on minimap but isn't available.

### ISSUE THREAD

**Issue by jim55price** [Questie/Questie#5082]

**Comment by jim55price** [comment]
The quest "Cormath the Courier" is showing on the minimap but isn't available at the questgiver.  This is a breadcrumb quest in the case of which I've already visited the target, an NPC in Camp Winterhoof.  The attached screenshot shows the NPC with no quest marker over his head, the popup expanded text for the quest in the minimap, & just peeking out from the bottom center of that popup, the very bottom of the "quest available" ! marker on the minimap.  After submitting this bug report, I'm hiding the minimap quest marker by clicking on the popup there.

![WoWScrnShot_090323_194514](https://github.com/Questie/Questie/assets/69993837/c6e49217-cf19-4fae-be7c-5ff427f42fc1)

**Comment by Cabro** [comment]
Cormath the Courier is the NPC. Quest in question is "Camp Winterhoof"

**Comment by jim55price** [comment]
Right, but this was addressed clearly & fully in my bug report.  You're 2/2
on telling me what I already mentioned, & 0/x on telling me anything that
might even remotely be described as a solution to this problem.  Don't let
it get you down, though, as I'm miles past that quest by now & only
reported it to you in the first place as a professional courtesy.  Cheers.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by BreakBB** [comment]
Hey @jim55price first of all thank you for your report, we are looking into finding the curl print and a solution for it.

Moreover I am quite sure @Cabro is only adding his two cents to this report, to make it easier for the team to find the solution 👍🏻

**Comment by jim55price** [comment]
I see.  In that case, perhaps I was a bit curt.  My apologies if any
offense was taken.

Let me add that this is not a new problem.  I found it very easily in an
old post on your website, so whatever fixed that (if anything ) probably
just ought to be replicated in this case.

Cheers.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by anon1231823** [comment]
@Cabro is this still an issue

**Comment by Cabro** [comment]
Yes
